src/middlewares/authenticate.js

```diff
@@ -0,0 +1,18 @@
+const expressJwt = require("express-jwt");
```

Comment:
Please document this file and explain the purpose of it and what its doing?

Reply:
Done